package-lock.json

```diff
@@ -5814,22 +5814,22 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": false,
```

Comment:
nit- did you intentionally include this file in this pr?

Reply:
Yes. Afaik, we either .gitignore it, or we should commit it everytime. The middle ground is inefficient. 

What do you think?

Reply:
i thought we were just periodically updating it, so that's why i asked. but i might very well be wrong, so up to you.